msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -199,7 +251,7 @@ public async Task<AuthenticationResult> AcquireTokenSilentAsync(IEnumerable<stri
         }
 
         /// <summary>
-        /// Removes all cached tokens for the specified account.
+        /// Removes all tokens in the cache for the specified account.
         /// </summary>
         /// <param name="account">instance of the account that needs to be removed</param>
         public async Task RemoveAsync(IAccount account)
```

Comment:
@henrik-me @shoatman @oldalton : do we all agree that Remove(IAccount) would better be named RemoveTokensForAccount? (I'd prefer personally)
since changing it later would be a breaking change, this might be the best moment to do it

cc: @SomkaPe @bgavrilMS @jennyf19 @trwalke @rayluo